meta-rpi-extras/recipes-qt/automotive/neptune-ui/neptune.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Neptune
+After=dbus-session@root.service
+Requires=dbus-session@root.service
+
+[Service]
+ExecStart=/usr/bin/appman -r -c /opt/am/sc-config.yaml -c am-config.yaml --dbus session Main.qml -platform eglfs
+Restart=on-failure
+WorkingDirectory=/opt/neptune
+Environment=HOME=/home/%u/
+Environment=QT_WAYLAND_HARDWARE_INTEGRATION=brcm
+Environment=LC_ALL=en_US
+Environment=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/root/dbus/user_bus_socket
```

Comment:
Isn't this line redundant?

Reply:
Ehm...no? This points out where the session bus is at, the `--dbus session` option only tells appman to use a session bus, not which one.

Reply:
Sure but systemd ususally does that automagically. Altough I now see that you specify root user